CONTRIBUTING.md

```diff
@@ -18,31 +18,44 @@ You can read our latest blog post on our
 
 To get started, simply follow these steps:
 
-   1. **Find a project:** review the projects listed under the
+1. **Find a project:** review the projects listed under the
    “Help Requested” list on the
    [Equity For Effort Projects Board](https://github.com/ParabolInc/parabol/projects/1)
    and select one you're interested in working on.
 
-   2. **Bid on it:** leave a comment on the project saying how many points
+2. **Bid on it:** leave a comment on the project saying how many points
    you’ll do the project for and we’ll move the project to the “Bid” stage
    (see: [Points and sizes](#points-and-sizes)) – if a project is unclear, you
    can ask for more clarification at this stage
 
-   3. **Do it:** once we've received your bid, we'll reach out via email and
+3. **Do it:** once we've received your bid, we'll reach out via email and
    verify you're qualified to participate in the E4E program. When that's
    out of the way we'll accept your bid and move the issue to the
    “Building” stage. You can then go ahead and perform the work. Once you’re
    done, submit a pull request and we'll move the project to the “Reviewing”
    stage.
 
-   4. **Collect points:** After the work has been reviewed and merged into the
+4. **Collect points:** After the work has been reviewed and merged into the
    source tree, we’ll update our public scoreboard and credit the creator(s)
    with the agreed upon number of points. Once you accrue more than
    100 points we issue options for equity in Parabol, Inc.
 
-# Git Strategy
-
-`git pull --ff-only` is recommended git pull stratagey if possible to keep the history as clean as possible.
+## Making changes
+
+- Fork the repository on GitHub.
+- Create a feature branch from where you want to base your work (this is usually the master branch).
+- Make commits of logical and atomic units.
+- Before committing, check for unnecessary whitespace with `git diff --check`.
+- Make sure your commit messages are in the proper format.
+- Use git pull strategy (`git pull --ff-only`) to keep the history as clean as possible during updating your local branch with changes from a tracked remote.
```

Comment:
+1 I'm not so sure this is the best advice. If you work together with someone else on the same feature branch, then this is the correct way, but usually there is only 1 author. In this case a more likely scenario is merging the base branch (usually master) into the feature branch. Then it is best to just `git merge origin/master`, it will complicate reviews otherwise.

Reply:
I've updated these steps.